dataprep/clean/clean_date.py

```diff
@@ -0,0 +1,996 @@
+            elif len(day_token) == 1:
+                result = result.replace(day_token, str("-"))
+        else:
+            if len(day_token) == 2:
+                if day < 10:
+                    result = result.replace(day_token, "0" + str(day), 1)
+                else:
+                    result = result.replace(day_token, str(day), 1)
+            else:
+                result = result.replace(day_token, str(day))
+    return result
+
+
+def transform_hms(
+    result_str: Union[str, Any],
+    hms_token: Union[str, Any],
+    ispm: Union[bool, Any],
+    hms_value: Union[int, Any],
+) -> Any:
+    """
+    This function transform parsed month into target format
+    Parameters
+    ----------
+    result_str
+        result string
+    hour_token
+        token of hour
+    hour
+        value of hour
+    """
+    result = deepcopy(result_str)
+    if hms_token != "":
+        if hms_value == -1:
+            if len(hms_token) == 2:
+                result = result.replace(hms_token, str("--"))
+            elif len(hms_token) == 1:
+                result = result.replace(hms_token, str("-"))
+        else:
+            if not ispm is None and ispm:
+                hms_value = hms_value - 12
+            if len(hms_token) == 2:
+                if hms_value < 10:
+                    result = result.replace(hms_token, "0" + str(hms_value), 1)
+                else:
+                    result = result.replace(hms_token, str(hms_value), 1)
+            else:
+                result = result.replace(hms_token, str(hms_value))
+    return result
+
+
+def transform_weekday(
+    result_str: Union[str, Any], weekday_token: Union[str, Any], weekday: Union[int, Any]
+) -> Any:
+    """
+    This function transform parsed month into target format
+    Parameters
+    ----------
+    result_str
+        result string
+    weekday_token
+        token of weekday
+    weekday
+        value of weekday
+    """
+    result = deepcopy(result_str)
+    if weekday_token != "":
+        if weekday == -1:
+            if len(weekday_token) == 3:
+                result = result.replace(weekday_token, str("---"))
+            elif len(weekday_token) == 5:
+                result = result.replace(weekday_token, str("-----"))
+        else:
+            if len(weekday_token) == 3:
+                result = result.replace(weekday_token, TEXT_WEEKDAYS[weekday - 1][0])
+            elif len(weekday_token) == 5:
+                result = result.replace(weekday_token, TEXT_WEEKDAYS[weekday - 1][1])
+    return result
+
+
+def transform_timezone(
+    result_str: Union[str, Any],
+    timezone_token: Union[str, Any],
+    timezone: Union[str, Any],
+    tz_info: Union[Dict[str, str], Any],
+) -> Any:
+    """
+    This function transform parsed month into target format
+    Parameters
+    ----------
+    result_str
+        result string
+    weekday_token
+        token of weekday
+    weekday
+        value of weekday
```

Comment:
The comment is not consistent with the parameters

Reply:
Refined.